test/integration/integration.h

```diff
@@ -160,6 +155,15 @@ class BaseIntegrationTest : Logger::Loggable<Logger::Id::testing> {
                                                Http::CodecClient::Type type);
   IntegrationTcpClientPtr makeTcpConnection(uint32_t port);
 
+  // Test-wide port map.
```

Comment:
When you say that bazel runs tests in parallel, I assume you mean in parallel in different processes, right? Otherwise it seems like this static port map is not going to work.